iocage

```diff
@@ -47,10 +47,10 @@ if [ "$(uname -K)" -lt "903000" ]; then
 fi
 
 # Check if libs are available
-if [ -e "/usr/local/lib/iocage" ] ; then
-    LIB="/usr/local/lib/iocage"
-elif [ -e "./lib" ] ; then
+if [ -e "./lib/ioc-common" ] ; then
     LIB="./lib"
+elif [ -e "/usr/local/lib/iocage" ] ; then
```

Comment:
Good question. Was the breakage before easy to reproduce without actually rebooting?

This *seems* pretty safe, as a nonexistent "./lib/ioc-common" would safely fail the if test, and should then give the elif a try. It would be good to confirm though for sure. I can't reboot my storage server right now to test though.

If the issue was reproducible some other way, I could certainly give it a shot.